data-products/tests/shared/unit/test_sqs.py

```diff
@@ -0,0 +1,33 @@
+import boto3
+import pytest
+from moto import mock_sqs
+from prefect import flow
+from shared.api_clients.sqs import RecommendationCandidate, put_results
+
+TEST_DATA = [RecommendationCandidate(feed_id=1, publisher="test", item_id=1)]
+
+
+@mock_sqs
+@pytest.mark.parametrize("test_config", [(False, 1), (True, 0)])
+def test_sqs(test_config):
+    curated, expiration = test_config
```

Comment:
This is how I use parametrize:
```suggestion
@pytest.mark.parametrize("curated,expiration", [(False, 1), (True, 0)])
def test_sqs(curated, expiration):
```
Just a different style. Up to you which seems better.